registry/auth.go

```diff
@@ -22,9 +21,8 @@ type challengeClient struct {
 }
 
 type registryEndpoint struct {
-	version   string
-	url       *url.URL
-	tlsConfig *tls.Config
```

Comment:
For here, could we just make it:
```
type registryEndpoint struct {
    url       *url.URL
    version   string
    // TODO: support tls for registry.
    // tlsConfig *tls.Config
}
```

Reply:
aggree that, by the way, can we just not do a lot of remove, I think maybe someone add this for use, @ZYecho @allencloud

Reply:
Thanks @Ace-Tang .
Could you please comment the field rather than removing it? @ZYecho